### PR TITLE
tests: Migrating the live node script to E2E

### PR DESCRIPTION
**Pull request by davidterpay**

### CODE REVIEW

**Comment by davidterpay** [comment]
Can we create a backport for main?

File: tests/e2e/e2e_queries_test.go
Comment (alexanderbez) [inline tests/e2e/e2e_queries_test.go:1]: These are all util functions right? I'd imagine a file named `e2e_queries_test.go` would have all tests related to queries, no?

File: tests/e2e/e2e_test.go [hunk 1]
```diff
@@ -1,7 +1,454 @@
-//go:build e2e
-
```
Comment (alexanderbez) [inline tests/e2e/e2e_test.go:2]: add this back :p

File: tests/e2e/e2e_queries_test.go
Comment (davidterpay) [reply tests/e2e/e2e_queries_test.go:1]: Good call. Moved the functions to utils.

**Comment by davidterpay** [comment]
@Mergifyio backport main

**Review by alexanderbez — APPROVED** [review]

**Pull request merged**